inc/boot.php

```diff
@@ -34,17 +36,75 @@
 define('DEBUG', true);
 
 
-
 /**
  * No special need to edit under this line
- * Except if it's a dev core
+ * Except if it's a dev core and working in a dev version or ...
+ */
+
+
+// if dev mod
+ini_set('display_errors', (int) DEBUG);
+if (DEBUG) {
+    error_reporting(-1);
+} else {
+    error_reporting(0);
+}
+
+
+
+/** [POC] log system
+ * What about logrotate ?
+ *
+ * enable log and set custom log path for PHP
+ *
+ * if you want to push an error message use with addionals informations use log_error('your message')
+ *                                          without                       error_log('your message')
+ *
+ * if you want to remove this POC, make sure to remove all log_error() in BT
+ */
+ini_set('log_errors', 1);
+// BT_root not defined at this point
+ini_set('error_log', dirname(dirname(__file__)).'/var/php-error.log');
+
+// global for show error
+$GLOBALS['errors'] = array();
+
+/** [POC]
+ * like error_log($message) but with addionals informations
+ * push in $GLOBALS['errors']
+ * can be used to only push in $GLOBALS['errors']
+ *
+ * ! if removed, take care, used in [core/addon] and maybe in other process ...
+ *
+ * @param string $message
+ * @param bool $write, write in log file
  */
+function log_error($message, $write = true)
+{
+    $GLOBALS['errors'][] = $message;
+
+    if ($write === true) {
+        $t = end((debug_backtrace()));
+        $log = ' v:'.BLOGOTEXT_VERSION.' in file '.$t['file'].' on line '.$t['line'];
+        error_log('BlogText: '.$message.$log);
```

Comment:
1. Étant donné qu'on est sur BT, pas besoin de le dire pour chaque erreur. La ligne peut commencer directement soit avec la version de BT diectement, soit avec l'erreur. Au choix.
2. Vu qu'on est des fous, on traduira `' in file '` et `' in file '`.
3. Ajoute des doubles quotes ou guillemets autour du nom du fichier.